presto-spi/src/test/java/com/facebook/presto/spi/resourceGroups/TestResourceGroupId.java

```diff
@@ -23,7 +27,7 @@
     @Test
     public void testBasic()
     {
-        new ResourceGroupId("test.test");
+        new ResourceGroupId("test_test");
```

Comment:
I meant that you should have an exception check - the test can have an expectedException annotation.